docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -4,12 +4,11 @@
 [id="hermetic-builds_{context}"]
 = Enabling hermetic builds
 
-Hermetic builds refer to a controlled build process in which a build system knows all the required resources and dependencies. This enhanced dependency visibility allows the build system to capture and produce a more precise record of the build's origin, dependencies, and modifications than it would otherwise be possible.
+A hermetic build is a secure, self-contained build process that doesn't depend on anything outside of the build environment. This means that it does not have network access, is not vulnerable to external influences, and cannot fetch dependencies at run time. Instead, you must declare all required resources and dependencies in your build definition. 
 
-In {ProductName}, hermetic builds are achieved by blocking network access to the build process. Therefore, you must pre-fetch all dependencies before the build can start.
-
-IMPORTANT: Hermetic builds disable network access, so the builds that require dependencies outside its Git repository could fail.
+In {ProductName}, you can block network access to the build process and run a hermetic build by setting the `*build-source-image*` parameter in your `image-controller-push.yaml` file to `true`. This means that you must fetch all dependencies _before_ the build can start.
```

Comment:
Is this section correct? I thought we were setting hermetic: true in the push and pull yamls? Does this need to be updated? We don't reference `build-source-image` elsewhere in this procedure.

Reply:
Yeah this should say `hermetic`, not `build-source-image`. Might be better to just link to the relevant procedure https://pr-201--boisterous-meerkat-894982.netlify.app/documentation/main/how-to-guides/proc_hermetic-builds/